src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Lottery {
+    private static final int MIN_BALL_NUMBER = 1;
+    private static final int MAX_BALL_NUMBER = 100;
+
+    public static Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
colorSupplier is your dependency here and it's better to initialize it in fields (or using constructor)